rlclientlib/CMakeLists.txt

```diff
@@ -227,3 +227,28 @@ install(TARGETS rlclientlib
   ARCHIVE DESTINATION lib
   LIBRARY DESTINATION lib
 )
+
+install(DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/
+    DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/rlclientlib
+    FILES_MATCHING
+    PATTERN "*.h"
+    PATTERN ".nuget" EXCLUDE
+    PATTERN "slim" EXCLUDE
+    PATTERN "win32" EXCLUDE
+    PATTERN "out" EXCLUDE
+    PATTERN "packages" EXCLUDE
+    PATTERN "x64" EXCLUDE
+    PATTERN ".vs" EXCLUDE
+)
+install(DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/../include

```

Comment:
The string view and future compat headers from vw are missing

Reply:
added